### PR TITLE
Change the format

### PR DESCRIPTION
**Pull request by BillWagner**
Respond to final feedback on PR #1077

When we merge this, do a merge commit, not a squash commit. We don’t want to lose the commit history from the previous PR.

### CODE REVIEW

**Comment by saldana** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tour-of-csharp/statements?branch=pr-en-us-1212](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tour-of-csharp/statements?branch=pr-en-us-1212)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C6%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611062018032052-1212%5Cworkflow_report.html).

File: docs/csharp/tour-of-csharp/statements.md [hunk 1]
```diff
@@ -38,79 +39,79 @@ The `using` statement is used to obtain a resource, execute a statement, and the
 
 The following lists the kinds of statements that can be used, and provides an example for each.
 
-***Local variable declaration***
+* Local variable declaration:
 
 [!code-csharp[Declarations](../../../samples/snippets/csharp/tour/statements/Program.cs#L9-L15)]
```
Comment (svick) [inline docs/csharp/tour-of-csharp/statements.md]: Written like this, each bullet point is its own list (`<ul>`), which is not semantically correct. To make it all one list, as it should be, each snippet should be indented by one space.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/csharp/tour-of-csharp/statements.md](https://github.com/BillWagner/docs/blob/change-the-format-1077/docs/csharp/tour-of-csharp/statements.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tour-of-csharp/statements?branch=pr-en-us-1212) | [Details](#user-content-docs/csharp/tour-of-csharp/statements.md)


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C7%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611071731146187-1212%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by BillWagner** [comment]
Build validated, so I'll squash my 2 commits into one.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/csharp/tour-of-csharp/statements.md](https://github.com/BillWagner/docs/blob/change-the-format-1077/docs/csharp/tour-of-csharp/statements.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tour-of-csharp/statements?branch=pr-en-us-1212) | [Details](#user-content-docs/csharp/tour-of-csharp/statements.md)


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C7%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611071746010344-1212%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by rpetrusha** [comment]
LGTM 
@BillWagner

**Comment by BillWagner** [comment]
Merged.  Thanks for contributing @badersur

**Pull request merged**